checker/jtreg/index/UnneededSuppressionsTest.out

```diff
@@ -0,0 +1,6 @@
+UnneededSuppressionsTest.java:9:1: compiler.warn.proc.messager: (unneeded.suppression)
+UnneededSuppressionsTest.java:22:5: compiler.warn.proc.messager: (unneeded.suppression)
```

Comment:
Is this for the `tainting` or the `lowerbound` suppression?
Can you separate the two to make sure that the `tainting` doesn't raise a warning when running the Index Checker.